src/lib/FreeC/Environment/Fresh.hs

```diff
@@ -114,3 +117,13 @@ freshCoqQualid :: String -> Converter Coq.Qualid
 freshCoqQualid prefix = do
   ident <- freshHaskellIdent prefix
   inEnv $ renameQualid ident
+
+-- | Generates a new Agda identifer based on the given name.
+--
+--   Note: Type is redundant with next merge from master and the introduction
+--   of @Fresh@
```

Comment:
Better use the `TODO` keyword in such comments instead of `Note`. I periodically search for `TODO` comments and resolve them.